nixos/modules/services/web-servers/h2o/vhost-options.nix

```diff
@@ -37,6 +37,15 @@ in
       '';
     };
 
+    host = mkOption {
+      type = types.nullOr types.nonEmptyStr;
+      default = null;
+      example = "127.0.0.1";
+      description = ''
+        Set the host address for this virtual host.
```

Comment:
Please link to the docs of the directive this sets, so people know what syntax is allowed here.